CesiumGS/cesium
# Add AEC design sandcastle examples

# Description

Implements three new Architecture, Engineering and Construction (AEC) related Sandcastle examples to support upcoming Cesium releases.

`AEC Architectural Design.html` demonstrates a typical architectural design model comprised of multiple tilesets with layer visibility toggles and metadata picking. 

`AEC Isolate by Category` demonstrates how to use architectural categories embedded within the tileset metadata to isolate building elements such as doors, floors, stairs, etc.

`AEC Metadata Styling.html` demonstrates how to interactively style pipeing layers, highlighting pipes whose length falls within a min / max range controller by a UI widget.

## Issue number and link

N/A

## Testing plan

1. Review each sandcastle example
2. Ensure tilesets load appropriately
3. Ensure UI elements and any interactive functionality are working
4. Make sure the title, description and thumbnail are correct

# Author checklist
